config/config.go

```diff
@@ -151,6 +152,11 @@ func Init(ctx context.Context, config Config, opts InitOptions) (*engine.Engine,
 		return nil, errors.New(errkind.Invalid, "no registries defined", op)
 	}
 
+
+	if config.TimeoutSeconds != nil && *config.TimeoutSeconds < 5 || *config.TimeoutSeconds > 3600 {
```

Comment:
Does this need to be a pointer? Why not just treat the zero value (literally 0 in this case lol) as meaning no timeout? Based on this validation 0 isn't even allowed as a valid value so why not use it for this?

I think this make make things simpler and this is pretty standard practice in Go. For example, see the `Timeout` field on `http.Client`: https://pkg.go.dev/net/http#Client

> A Timeout of zero means no timeout.

Reply:
that's a good point, though is there a way to know if someone explicitly set timeoutSeconds: 0 or if it's just not provided in .tbrc.yml? i'm asking because we're defaulting to 3600 when the value is not provided and i'm not sure if there is a way to distinguish between it not being set or it being set to 0

Reply:
There isn't any way to distinguish, someone could put `timeoutSeconds: 0` in their `.tbrc.yml` but I guess the question is does it matter? I don't see why anyone would do that, and even if they did I don't think it matters, the behaviour should be the same